options/ansi/generic/string-stubs.cpp

```diff
@@ -485,3 +485,17 @@ void *memmem(const void *, size_t, const void *, size_t) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+// BSD extension.
+// Taken from musl
+size_t strlcpy(char *d, const char *s, size_t n) {
```

Comment:
Please add a test.

Reply:
Done, also added `strlcat` and tests for that as well.